examples/react-native/README.md

```diff
@@ -193,3 +193,9 @@ If the example app is not picking up changes to the values in _.env_ close Metro
 ```bash
 yarn react-native-example dev --reset-cache
 ```
+
+### Andoid emulator fails to connect to Metro server
+
+```bash
+`adb reverse tcp:8081 tcp:8081`
```

Comment:
Since the other commands in this file aren't wrapped backticks might make sense to omit them here for consistency?

Reply:
Good point, standardized in both readmes.